packages/unmock-cli/src/commands/list.ts

```diff
@@ -0,0 +1,59 @@
+import { readFileSync } from "fs";
+import * as glob from "glob";
+import * as path from "path";
+import { ILogger } from "unmock-core";
+import WinstonLogger from "../utils/logger";
+
+const removeTrailingNumber = (s: string) =>
+  s
+    .split(" ")
+    .slice(0, -1)
+    .join(" ");
+
+const SNAP_SUFFIX = ".snap";
+const DEFAULT_SNAPSHOTS_FOLDER = ".snapshots";
+interface IStringMap {
+  [key: string]: string[];
+}
+
+export const listInternal = (logger: ILogger): Promise<undefined> => {
```

Comment:
Not sure how relevant this is anymore...

Reply:
That makes two of us.